.gitignore

```diff
@@ -7,6 +7,9 @@ __pycache__/
 # C extensions
 *.so
 
+# VSCode settings
+.vscode
```

Comment:
what does this do?

Reply:
ignores a folder created by vscode. This folder contains your personal vscode settings and configurations.